conans/client/graph/graph.py

```diff
@@ -229,6 +231,29 @@ def serialize(self):
         result["test"] = self.test
         return result
 
+    def package_url(self):
+        """
+        Creates a PURL following https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst
```

Comment:
read the spec: https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#conan

Reply:
Do you want me to change the comment, or is there an error in how I build the JSON?

Reply:
wanted to put a more precise link here, so nobody needs to scroll trough the whole document.